spec/install/gemfile/path_spec.rb

```diff
@@ -13,6 +13,21 @@
     should_be_installed("foo 1.0")
   end
 
+  it "resolves from prior lockfile for source path gems" do
+    build_lib "rack-obama", "1.0", :path => lib_path("same_path")
+    build_lib "net-ssh", "1.0",    :path => lib_path("same_path")
+
+    install_gemfile <<-G
+      path "#{lib_path("same_path")}/" do
+        gem 'rack-obama'
+        gem 'net-ssh'
```

Comment:
Do I need to cleanup these gems and paths when I'm done?  I'm afraid I might be introducing some global state causing other tests to fail.  Maybe you have some ideas?